Makefile

```
@@ -1,19 +1,25 @@
-.PHONY: help lint test test-verbose test-coverage test-coverage-html test-clean
+.PHONY: help lint test-all test-verbose test-coverage test-coverage-html test-clean install-pre-push-hook uninstall-pre-push-hook
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)